core/src/main/scala/akka/persistence/r2dbc/internal/Sql.scala

```diff
@@ -41,7 +42,11 @@ object Sql {
       var i = 0
       while (i < sql.length) {
         val c = sql.charAt(i)
-        if (c == '?') {
+        val d = if (i < sql.length - 1) sql.charAt(i + 1) else 0
```

Comment:
Perhaps this could be a
```
def isNext(d: Char): Boolean =
  (i < sql.length - 1) && (sql.charAt(i + 1) == d)
```
then next line can be
```
if (c == '?' && isNext('?')) {
```